### ISSUE TITLE
IE Only - Data Tables Sort Arrows for Keyboard users

### ISSUE THREAD

**Issue by ltsheu** [18F/dol-whd-14c#544]
> **Note:** This would be a Data Tables / Angular task.

**IE specific Data Tables bug (found by OCIO 508 tester):**

The sorting (up and down arrows) for Worksite and Employee tables does not work. **They sort only once. Subsequent sorts are not available for keyboard users in IE (works in Chrome).**

**Comment by binwang89** [comment]
@davoodharun please provide the estimate for this issue. Thanks

**Comment by mmurthydol** [comment]
Harun mentioned the fix is complicated and mostly a hack.
